ui/app/templates/vault/cluster/access/mfa/methods/create.hbs

```diff
@@ -0,0 +1,72 @@
+<PageHeader as |p|>
+  <p.levelLeft class="has-border-bottom-light">
+    <h1 class="title is-3">
+      {{#if this.method}}
+        Configure
+        {{this.method.name}}
+        MFA
+      {{else}}
+        Multi-factor authentication
+      {{/if}}
+    </h1>
+  </p.levelLeft>
+</PageHeader>
+
+<div class="has-border-top-light">
+  {{#if this.method}}
+    {{! display forms }}
+  {{else}}
+    <p class="has-top-margin-l">
+      Multi-factor authentication (MFA) allows you to set up another layer of security on top of existing authentication
+      methods. Vault has four available methods.
+      <a href="https://www.vaultproject.io/api-docs/secret/identity/mfa" target="_blank" rel="noopener noreferrer">Learn
```

Comment:
You can use the DocLink component. Example for one that links to the API docs is in swagger-ui. `<DocLink @path="/api-docs#namespaces">docs</DocLink>`

Reply:
Nice that's great! Thanks for pointing that component out. I've updated those links.